exercises/practice/word-count/.meta/config.json

```diff
@@ -1,6 +1,14 @@
 {
   "blurb": "Given a phrase, count the occurrences of each word in that phrase.",
-  "authors": [],
+  "authors": [
+    "kabiir"
+  ],
+  "contributors": [
+    "devkabiir",
```

Comment:
@kabiir is @devkaviir, I believe. @devkabiir is this true?

Reply:
Yes!. And please use `@kabiir`, so they don't get annoyed. GitHub made my old username available as a valid username after some time, which happened to be taken by someone.

Reply:
> And please use @kabiir, so they don't get annoyed. 

Use kabiir and not devkabiir? But the current user `kabiir` did not do the work, you did.

Reply:
I meant that we should use backticks like ``` to mention the user `kabiir` instead of directly sending them a ping when using `@`kabiir.